dts/arm/st/h7/stm32h745.dtsi

```diff
@@ -39,6 +39,19 @@
 			status = "disabled";
 		};
 
+		usbotg_hs: usb@40040000 {
+			compatible = "st,stm32-otghs";
+			reg = <0x40040000 0x40000>;
```

Comment:
Is it an other controller on the SoC or why it has a different `reg = <0x40040000 0x40000>;` ?